runtime/src/bank.rs

```diff
@@ -3627,6 +3621,12 @@ impl Bank {
                         }
                     }
                 }
+            } else {
```

Comment:
What do you think about this as a way to consolidate existing epoch dependent features into FeatureSet?  Doing it this way means the feature would turn on at some epoch or we could enable it at anytime before.